src/core/bitop.d

```diff
@@ -706,3 +706,38 @@ version (D_InlineAsm_X86_64)
         }
     }
 }
+
+/**
+ *  Bitwise rotates unsigned arithmetic types
+ */
+pure T rotl(T)(in T u, in uint s)
```

Comment:
I'd prefer `n` or `count` instead of `s`, and `val` or `value` instead of `u`.
Any particular reason for naming them `u` and `s`?